cmd/minikube/cmd/stop.go

```diff
@@ -65,7 +65,6 @@ func init() {
 // runStop handles the executes the flow of "minikube stop"
 func runStop(cmd *cobra.Command, args []string) {
 	out.SetJSON(outputFormat == "json")
-	register.SetEventLogPath(localpath.EventLog(ClusterFlagValue()))
```

Comment:
lets check if if the path exists, then set it, so we still write the events to the event log

Reply:
Makes sense @medyagh. Thanks.

Reply:
Question: Is there a case where a profile folder exists without event.json file in it. I believe it exists most of the time but still the SetEventtLogPath function creates one if it doesn't exists which means some corner cases.